app/views/admin/shared/tagging/_legacy_associations_box.html.erb

```diff
@@ -0,0 +1,46 @@
+<section class="legacy-associations" id="legacy-associations">
```

Comment:
The class and id aren't being used, we could probably remove them (or at least the id)